### ISSUE TITLE
Add a configurable delay on scores.

### ISSUE THREAD

**Issue by saxonpaul92** [riffnshred/nhl-led-scoreboard#133]
Love the scoreboard.  Very cool especially to hockey fans!

My request is very small.

When running nhl_setup add a feature that allows the end user to add a delay (in seconds) that will give the user the chance to sync up the delay.

Since each broadcast has different delays this will give each user the opportunity to bridge the gap.

Of course this only works  when the broadcast adds a delay. 

Thankx

**Comment by riffnshred** [comment]
Thanks bud !!

This has been an issue since the very beginning. 
Duplicate of #6 

It's kinda complicated and though I could just put a delay in there for every request to the API. But obviously it does not work like that. I sorted figured how to do it, but the amount of complexity multiply throughout the scoreboard and without being sure that it will be stable. I'm always close to figure something simple and realize why working on it that it will cause other issues down the line.

Funny enough, last night I think I figured a simple way to just cache the goal when there is a new one and show it only after the amount of delay set by the user has passed. If that works, I'll have it for an upcoming update.
